docs/deployment/getting-started.md

```diff
@@ -0,0 +1,76 @@
+# Deployment
+
+Server deployment is a complex area, that will depend on what kind of service you're deploying Uvicorn onto.
+
+### TL;DR
+As a general rule, you probably want to:
+
+* Run `uvicorn --reload` from the command line for local development.
+* Run `gunicorn -k uvicorn.workers.UvicornWorker` for production (non containerized).
+* Run behind Nginx for self-hosted deployments.
+* Finally, run everything behind a CDN for caching support, and serious DDOS protection.
```

Comment:
```suggestion
```
what does everything mean ? I'd remove that, it's too generic, not actionnable

Reply:
Most of these lines were from current documentation. The deployments.md file.

My initial thought was to replace that with getting-started.md and add section wise files.